modules/sql-engine/src/integrationTest/sql/join/natural/test_natural_join.test

```diff
@@ -48,24 +46,28 @@ SELECT * FROM t1 NATURAL JOIN t3
 ----
 1	2	3
 
-query III
-SELECT a, b, c FROM t3 NATURAL JOIN t2
-----
-1	2	3
-
 # common columns moved to the first place, column order: a, c, b
 query III
 SELECT * FROM t3 NATURAL JOIN t2
 ----
 1	3	2
 
+# skipif ignite3
```

Comment:
why it skipped? we have the same types for all tables

Reply:
Thanks. Uncommented skipif instruction.